### DISCUSSION TITLE
Getting client status in zabbix

### DISCUSSION THREAD

**Discussion by Smiley-k** [rustdesk/rustdesk#5787] [Q&A]
Could you tell me how I can get from the server to my monitoring system the statuses of my clients (androdi\windows\linux) when they are online or have launched access for me ? 

I would like to retrieve data either from the database or web service online (offline) whether the connection is currently online or not

**Comment by rsdenck** [comment]
Sim! Pode sim! Seu servidor RustDesk OSS é Linux ou Windows? Qualquer um deles é possível!
